### ISSUE TITLE
Load model trained on Detectron framework

### ISSUE THREAD

**Issue by Atul997** [open-mmlab/mmdetection#3835]
Hi, 

Can we load a model trained on the Detectron framework and start finetuning in MMdetection v2.x.x?
If so, then how to approach it? Please suggest.

**Comment by ZwwWayne** [comment]
Yes you can. MMDetection is fully compatible with Detectron2. You only need to map the keys and weights to the format of MMDetection. [This script](https://github.com/xvjiarui/mmdetection/blob/detectron2_convert/tools/detectron2pytorch.py) provides an example to convert Mask R-CNN trained by detectron2 to mmdet format.  The inference accuracy should be similar.

**Comment by Atul997** [comment]
Thanks, @ZwwWayne  I will give a shot to this.

**Comment by vealocia** [comment]
@Atul997 Hi, did you find the correct way to convert detectron2 pretrained model to MMDetection formation? Can you share your convert scripts, great thanks!

**Comment by Keiku** [comment]
@vealocia Did you use the above script after all? For what purpose did you want to do it?
